DEVELOP.md

```diff
@@ -204,18 +150,18 @@ Their upstream repos are assumed to have the remote name `origin`.
       - `git checkout -b stable_$MN`
       - `git push --set-upstream origin stable_$MN`
 
-    * Log on to [RTD](https://readthedocs.org/), go to the `pywbem` project,
+    * Log on to [RTD](https://readthedocs.org/), go to the `pywbemtools` project,
       and activate the new branch `stable_M.N` as a version to be built.
 
-18. On GitHub, edit the new tag, and create a release description on it. This
+14. On GitHub, edit the new tag, and create a release description on it. This
     will cause it to appear in the Release tab.
 
-19. On GitHub, close milestone `M.N.U`.
+15. On GitHub, close milestone `M.N.U`.
 
     Note: Issues with that milestone will be moved forward in the section
     "Starting a new version".
 
-20. Upload the package to PyPI:
+16. Upload the package to PyPI:
```

Comment:
Question. Is there a means to test upload to PyPi?

Second, does that mean that if there is an issue with the upload we have to go to new minor version?

Reply:
Testing pypi uload is not possible, to my knowledge.
Yes.